### PR TITLE
cook schema

### PR DESCRIPTION
**Pull request by Eng-NUREDDIN**

### CODE REVIEW

File: src/models/cookSchema.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  cook_phone: {
+    type: String,
+    required: true,
+  },
+  cook_info: {
+    type: String,
+  },
+  cook_name: {
+    type: String,
+  },
```
Comment (radmanlo) [inline src/models/cookSchema.js:17]: I think it is a good idea we have created_at attribute here in order to find out when it is created (Most probably I forgot to put in the design).

File: src/models/cookSchema.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  cook_phone: {
+    type: String,
+    required: true,
+  },
+  cook_info: {
+    type: String,
+  },
+  cook_name: {
+    type: String,
```
Comment (radmanlo) [inline src/models/cookSchema.js:16]: it better that cook_name be required also

File: src/models/cookSchema.js (hunk 1 above)
Comment (radmanlo) [inline src/models/cookSchema.js:17]: I think it is a good idea we have created_at attribute here in order to find out when it is created (Most probably I forgot to put in the design).

File: src/models/cookSchema.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  },
+});
+
+module.exports = mongoose.model('Cook', cookSchema);
```
Comment (radmanlo) [inline src/models/cookSchema.js:20]: Since we have cook as router it is good idea that  Coolk be CookSchema

**Review by radmanlo — CHANGES_REQUESTED** [review]
Please change the code based on the comments I write. Thank you

**Review by radmanlo — APPROVED** [review]

**Review by afrakucukaydin — APPROVED** [review]

**Review by talal-bakkour — APPROVED** [review]

**Pull request merged**